tools/harness-thci/OpenThread.py

```diff
@@ -2322,10 +2322,6 @@ def startNativeCommissioner(self, strPSKc='GRLPASSPHRASE'):
         print(cmd)
         return self.__executeCommand(cmd)[-1] == 'Done'
 
-    @API
-    def getBorderAgentPort(self):
```

Comment:
We don't need this API anymore?

Reply:
We don't since it's always possible to inquiry a Border Router for this parameter via mDNS.